middlewares/ratingValidation.js

```diff
@@ -0,0 +1,44 @@
+import { isNumeric } from 'validator';
+
+export default {
+  verifyMentor(req, res, next) {
```

Comment:
The variable name here could better capture what it is validating. For example, `userIsMentor`